Emby.Server.Implementations/LiveTv/Listings/SchedulesDirect.cs

```diff
@@ -115,9 +113,7 @@ public async Task<IEnumerable<ProgramInfo>> GetProgramsAsync(ListingsProviderInf
                     Url = ApiUrl + "/programs",
                     UserAgent = UserAgent,
                     CancellationToken = cancellationToken,
-                    LogErrorResponseBody = true,
-                    // The data can be large so give it some extra time
-                    TimeoutMs = 60000
```

Comment:
why?

Reply:
I see why drop timeout (like "not supported"), but why drop logging error response?

Reply:
The property was being set, but is wasn't actually being used iirc.